Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UIBackground/UIBackgroundInstantiationSystem.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.Optimization.PerformanceBudgeting;
+using DCL.Optimization.Pools;
+using DCL.SDKComponents.SceneUI.Classes;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Groups;
+using DCL.SDKComponents.SceneUI.Utils;
+using ECS.Abstract;
+using ECS.Prioritization.Components;
+using ECS.StreamableLoading.Common.Components;
+using ECS.StreamableLoading.Textures;
+using ECS.Unity.Textures.Components;
+using ECS.Unity.Textures.Components.Defaults;
+using SceneRunner.Scene;
+using UnityEngine;
+using Promise = ECS.StreamableLoading.Common.AssetPromise<UnityEngine.Texture2D, ECS.StreamableLoading.Textures.GetTextureIntention>;
+
+namespace DCL.SDKComponents.SceneUI.Systems.UIBackground
+{
+    [UpdateInGroup(typeof(SceneUIComponentInstantiationGroup))]
+    [LogCategory(ReportCategory.SCENE_UI)]
+    public partial class UIBackgroundInstantiationSystem : BaseUnityLoopSystem
```

Comment:
The flow of clean-up and reconfiguration looks incorrect:
- `TryCreateGetTexturePromise` is called for `new UIBackgroundComponent()` only: `uiBackgroundComponent.TexturePromise` is always null. 
- Thus, when `image URL` is changed Promise is never released and creates a leak
- `UIBackgroundReleaseSystem` does not release the underlying promise: it releases `DCLImage` only

Reply:
You were right, I was not taking into account the texture's url updating from the scene. I've re-implemented the flows in order to fix it, also to manage a `LifeCycle` status for the textures loading and manage the promises correctly.

@mikhail-dcl could you please take a look? I'ts done here: https://github.com/decentraland/unity-explorer/pull/259/commits/de3005c02b8f49e21acd33a7b4614eb3fb4411f0